sample/benchmark/src/main/java/com/datadog/benchmark/sample/fragment/SessionReplayAppcompatFragment.kt

```diff
@@ -20,7 +20,7 @@ import android.widget.ImageView
 import android.widget.NumberPicker
 import android.widget.Spinner
 import androidx.fragment.app.Fragment
-import androidx.navigation.fragment.NavHostFragment.findNavController
+import androidx.navigation.fragment.NavHostFragment.Companion.findNavController
```

Comment:
is this change coming from navigation dependency update brought by the Compose BOM?

Reply:
Yes exactly, it seems there is an override version of navigation lib due to the compose bom import.